lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -553,9 +554,10 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
     course = get_course_by_id(course_id)
     cohorts_header = ['Cohort Name'] if course.is_cohorted else []
 
-    partition_service = LmsPartitionService(user=None, course_id=course_id)
-    partitions = partition_service.course_partitions
-    group_configs_header = ['Group Configuration Group Name ({})'.format(partition.name) for partition in partitions]
+    experiment_partitions = get_split_user_partitions(course.user_partitions)
```

Comment:
Nit: why do we have to have 3 different names here ("experiment_partitions", "split_user_partitions", and "Group Configuration" on line 559). I'd recommend at least ditching experiment_partitions and call it split_user_partitions to be consistent with the existing method name.